webrtc.html

```diff
@@ -5826,7 +5826,8 @@ <h2>Methods</h2>
                     <li>
                       <p>Determine if negotiation is needed to transmit
                       <var>withTrack</var> in place of the sender's existing
-                      track. Negotiation is not needed if <var>withTrack</var>
+                      track. Negotiation is not needed if no codec changes
```

Comment:
Do we want the word "envelope" in there?
ie changing from 640x480 to 320x240 doesn't require changes if the negotiated limits are 0x0 to 1024x768.